Khan/live-editor
# Tweak image resizing script and add new images

I changed the script so that when I call it with no arguments,
it just Does The Right Thing (makes original and thumbs).
I also added more images, inc. 2 new categories, landscapes and food.
I could actually remove /images from the build folder now,
as I'm uploading them to S3 now.
Let me know if you think I should do that now.

Test Plan:
- Go to /cs/new/webpage
- Insert an <img>, see new food and landscapes categories show up
